modules/virt-creating-vm-instancetype.adoc

```diff
@@ -58,27 +61,54 @@ Logos of operating systems that are not available in the cluster are shown at th
 +
 In addition, there is a link to the *Create a Windows boot source* quick start. The same link appears in a popover if you hover the pointer over the question mark icon next to the _Select volume to boot from_ line.
 +
-Immediately after you install the environment or when the environment is disconnected, the list of volumes to boot from is empty. In that case, three operating system logos are displayed: Windows, {op-system-base}, and Linux. You can add a new volume that meets you requirements by clicking the *Add volume* button.
+Immediately after you install the environment or when the environment is disconnected, the list of volumes to boot from is empty. In that case, three operating system logos are displayed: Windows, {op-system-base}, and Linux. You can add a new volume that meets your requirements by clicking the *Add volume* button.
+ifdef::dynamic-key[]
+* Click *Add volume* to upload a new volume or use an existing persistent volume claim (PVC), volume snapshot, or data source. Then click *Save*.
+endif::[]
+ifdef::virt-create-vms[]
+* Click *Add volume* to upload a new volume or use an existing persistent volume claim (PVC), volume snapshot, or data source. Then click *Save*.
+endif::[]
 
 ifdef::virt-create-vms[]
 . Click an instance type tile and select the resource size appropriate for your workload.
 endif::[]
 ifdef::dynamic-key[]
 . Click the *Red Hat Enterprise Linux 9 VM* tile.
 endif::[]
-. If you have not already added a public SSH key to your project, click the edit icon beside *Authorized SSH key* in the *VirtualMachine details* section.
-. Select one of the following options:
+ifdef::virt-create-vms[]
+. (Optional) Choose the VirtualMachine details, including the VM's name, that apply to the volume you are booting from.
+endif::[]
+ifdef::virt-create-vms[]
+** For a Linux-based volume, follow these steps to configure SSH:
+endif::[]
+
+.. If you have not already added a public SSH key to your project, click the edit icon beside *Authorized SSH key* in the *VirtualMachine details* section.
+.. Select one of the following options:
 
 * *Use existing*: Select a secret from the secrets list.
 * *Add new*:
```

Comment:
Outside of the scope of the PR, but this seemed strange just having `*Add new*:` as the text leading into substeps. Also I don't think it's rendering / intending as intended:
<img width="536" alt="image" src="https://github.com/openshift/openshift-docs/assets/3442316/22aa8d99-0ab9-415b-b171-2ea2956ac8e0">

Maybe this, which includes use of [open blocks](https://docs.asciidoctor.org/asciidoc/latest/blocks/open-blocks/) to help keep things together / aligned properly:
```
* *Add new*: Follow these steps:
+
--
.. Browse to the public SSH key file or paste the file in the key field.
.. Enter the secret name.
.. Optional: Select *Automatically apply this key to any new VirtualMachine you create in this project*.
.. Click *Save*.
--
```
But of course, apologies if my read on what's going on here is off! 😅